pyproject.toml

```diff
@@ -134,6 +134,7 @@ npm = [
 [tool.jupyter-releaser]
 skip = [
     "check-links",
+    "check-wheel-contents",
 ]
 
 [tool.jupyter-releaser.hooks]
```

Comment:
We'll also need to update the name of the `after-draft-release` hook.

Need to double check which one to use.